recipes/conda/meta.yaml

```diff
@@ -0,0 +1,65 @@
+{% set version = "4.1.5" %}
+
+package:
+  name: conda
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/conda/conda
+  git_tag: {{ version }}
+  git_depth: 1
+
+build:
+  number: 0
+  script:
+    - unlink "${PREFIX}"/bin/conda                  # [unix]
+    - CONDA_DEFAULT_ENV='' python setup.py install
+  entry_points:
+    - conda = conda.cli:main
+  always_include_files:
+    - bin/conda                 # [unix]
+    - bin/activate              # [unix]
+    - bin/deactivate            # [unix]
+    - Scripts/activate.bat      # [win]
+    - Scripts/deactivate.bat    # [win]
+
+requirements:
+  build:
+    - git
+    - python
```

Comment:
Guessing I pushed when you commented. Sorry for the discontinuity. Yeah, that makes sense. We only recently added the later (i.e. a couple of minutes ago). 😄